Middleware/SVGElements/Text.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+public class Text
+{
+    public Text() 
+    {
+
+    }
+
+    public async Task<string> SVG(string wrapper, IWebHostEnvironment _env, string textX, string textY, string fontWeight, string fontSize, string textColor, string fill, string textDecoration, string opacity, string text, string animations) 
```

Comment:
I would consider a builder pattern here since this is a lot of parameters and there might be reasonable defaults for many of them.